api/Document.json

```
@@ -8901,6 +8901,53 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "accepts_TrustedHTML": {
+          "__compat": {
+            "description": "Accepts `TrustedHTML` instances",
```

@wbamberg write and writeln() modified as suggested, based on the fact that these take just one parameter (actually they take multiple parameters of string and/or TrustedHTML, represented as `...text` in the spec).

I think technically the right way to do this according to the schema is as shown, noting that markup _will_ be used in the docs

```suggestion
        "markup_param_accepts_TrustedHTML": {
          "__compat": {
            "description": "Accepts `TrustedHTML` instances",
```

---

Agree that as these only support a single parameter, there is no need to over-complicate this.